app/models/planning_application.rb

```diff
@@ -272,10 +272,6 @@ def application_number
     self[:application_number].to_s.rjust(5, "0")
   end
 
-  def reviewer_disagrees_with_assessor?
-    to_be_reviewed?
-  end
```

Comment:
This one hasn't been used in four years either, and if it still was I'd probably just replace it with `to_be_reviewed?` anyway.